README.md

```diff
@@ -855,12 +855,13 @@ Code Formatters
 
 ## Package Repositories
 
-*Local PyPI repository server and proxies.*
+*Local and hosted PyPI repository server and proxies.*
 
 * [warehouse](https://github.com/pypa/warehouse) - Next generation Python Package Repository (PyPI).
 * [bandersnatch](https://github.com/pypa/bandersnatch/) - PyPI mirroring tool provided by Python Packaging Authority (PyPA).
 * [devpi](https://github.com/devpi/devpi) - PyPI server and packaging/testing/release tool.
 * [localshop](https://github.com/jazzband/localshop) - Local PyPI server (custom packages and auto-mirroring of pypi).
+* [cloudsmith :heavy_dollar_sign:](https://cloudsmith.io/l/python-repository/) - A fully managed package management SaaS, with first-class support for public and private Pyhton repositories (plus many others). Has a generous free-tier and is also completely free for open-source.
```

Comment:
small typo here - should read Python rather than Pyhton.

Reply:
Good spot!